tf/modules/monitoring/metric-alarms.tf

```diff
@@ -1,28 +1,26 @@
 locals {
   alarms = {
-    "es-cluster-health-${var.environment}" = {
-      display_name            = "Elasticsearch Cluster Health Status"
-      filter                  = "metric.type = \"prometheus.googleapis.com/elasticsearch_cluster_health_status/gauge\" AND metric.labels.color = \"green\""
-      comparison              = "COMPARISON_LT"
+    "es-cluster-yellow-${var.environment}" = {
+      display_name            = "Elasticsearch Cluster Health Yellow"
+      filter                  = "metric.type = \"prometheus.googleapis.com/elasticsearch_cluster_health_status/gauge\" AND metric.labels.color = \"yellow\""
+      comparison              = "COMPARISON_GT"
       evaluation_missing_data = "EVALUATION_MISSING_DATA_ACTIVE"
       trigger_count           = 1
-      threshold_value         = 1
-      duration                = "60s"
+      threshold_value         = 0
+      duration                = "900s"
       condition_absent        = "300s"
       min_group_by            = "metric.label.es_cluster"
```

Comment:
I was somewhat struggling to confirm my suspicion but I suspect after inverting the `COMPARISON_LT` to `COMPARISON_GT` this should become `max_group_by` (and same logic for the `red` check)

Reply:
Look's like the provider has undergone a major version bump beyond where we are at (5.29.1) and then looking here https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/monitoring_alert_policy is seems like the format may have changed somewhat in general for grouping to become more standard with the other apis and we might now need to us the https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/monitoring_alert_policy#cross_series_reducer for this logic.